pallets/cash/src/chains.rs

```diff
@@ -348,72 +394,138 @@ impl FromStr for ChainId {
 #[derive(Clone, Eq, PartialEq, Encode, Decode, RuntimeDebug, Types)]
 pub enum ChainBlock {
     Eth(<Ethereum as Chain>::Block),
+    Matic(<Polygon as Chain>::Block),
```

Comment:
kind of funny to have call it Matic and Polygon on the same line 🤔

Reply:
yeah "matic" is the ticker, "matic" is to Polygon as "eth" is to Ethereum